zoom_monitor.py

```diff
@@ -108,6 +109,19 @@ async def _check_if_meeting_ended(self):
             self._meeting_ended = True  # Don't try logging out later
             raise MeetingEndedException()
 
+    async def _click_child_button(self, parent: str, child: str, child_role="button"):
+        parent_button = self._driver.get_by_role("button", name=parent)
+        for attempt in range(4):
+            try:
+                await parent_button.dblclick()
+                child_button = self._driver.get_by_role(child_role, name=child)
+                await child_button.click(timeout=100)
+                break
+            except TimeoutError as e:
+                if attempt == 4:
```

Comment:
Mild preference to put a local variable at the top of this function for `maximum_attempts`. That way, if we want to change from 4 to something else, we don't need to change both this line and line 114.